pkg/applyconfigurations/testdata/cronjob/applyconfiguration/internal/internal.go

```diff
@@ -0,0 +1,46 @@
+// Code generated by applyconfigurations. DO NOT EDIT.
```

Comment:
Having applyconfiguration and testapplyconfiguration looks like have the golden files twice?

Do we have to check these in twice? Or can we e.g. get away with having one of these folders on .gitignore?